simplq/src/components/common/Modal/Modal.jsx

```diff
@@ -1,9 +1,8 @@
 import React from 'react';
-import Modal from '@material-ui/core/Modal';
-import styles from './Modal.module.scss';
+import { Dialog } from '@material-ui/core';
 
 export default ({ open, onClose, children }) => (
-  <Modal className={styles['root']} open={open} onClose={onClose}>
-    <div className={styles['content']}>{children}</div>
-  </Modal>
+  <Dialog PaperProps={{ style: { width: '27%', borderRadius: 30 } }} open={open} onClose={onClose}>
```

Comment:
Is it possible to define the CSS in `./Modal.module.scss` and use it here? We want to define styles separately in .scss files wherever possible.

Reply:
Yes it is, Actually the main purpose of *.module.scss is that we can define CSS module inside SCSS file which makes it different from a normal *.scss files

Reply:
Yeah Actually it is possible, i will modify it now